nest-forms-backend/prisma/schema.prisma

```diff
@@ -66,7 +66,7 @@ enum GinisState {
 enum FormState {
   DRAFT
   QUEUED
-  SENDING_TO_NASES
+  SENDING_TO_NASES // no longer used, we go directly from DRAFT/QUEUED to DELIVERED_NASES
```

Comment:
We probably can, but since this is db level, we'd also need a migration that clears up any that are presently in that state. Would create as a separate task.